newIDE/app/src/GameDashboard/GameAnalyticsPanel.js

```diff
@@ -142,6 +144,19 @@ export const GameAnalyticsPanel = ({
                 <Column noMargin alignItems="center" expand>
                   <Text size="block-title" align="center">
                     <Trans>{chartData.overview.playersCount} sessions</Trans>
+                    <Tooltip
+                      title={
+                        <Text>
+                          <Trans>
+                            Number of people who launched the game. Viewers are
+                            considered players when they stayed at least 60
+                            seconds including loading screens.
+                          </Trans>
```

Comment:
Tooltips must as short as possible.
```suggestion
                          <Trans>
                            Viewers are
                            considered players when they stayed at least 60
                            seconds including loading screens.
                          </Trans>
```

Reply:
The tooltip must define the graph title and the graph content.
I can't reduce this description, it may be obvious to you, but it's not the case for everyone, which is why this PR was opened.

Reply:
Not necessarily, a tooltip can be a way to give a small precision. The same way as the tips in the wiki.